doc/service-agent-api.md

```diff
@@ -170,13 +170,12 @@ Parameters:
 
 By sending this message, a Verifiable Credential is effectively revoked and a notification is sent to the DIDComm connection it has been issued to.
 
-This message could be sent as a response to a Credential issuance. In such case, `connectionId` and `revocationDefinitionId` is used to identify credential details.
+This message could be sent as a credential revocation notification. In such case, `threadId` is used to identify credential details.
```

Comment:
Remove or reformulate the entire paragraph. It made sense in the message you are basing this section, but here the first sentence is basically repeating the same thing as the previous one.

Reply:
fixed